addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/InfoPrivateAddressDisclosureScanRule.java

```diff
@@ -1,33 +1,22 @@
 /*
+ * Zed Attack Proxy (ZAP) and its related class files.
  *
- * Paros and its related class files.
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
```

Comment:
DOH I guess I need to change the spotless exceptions :wink:

Will tackle that in a bit and restore the relevant bits.